java-tools/OsmAndServer/src/main/java/net/osmand/server/controllers/user/MapApiController.java

```diff
@@ -592,6 +592,7 @@ public ResponseEntity<String> getAccountInfo() {
 		final String EXPIRE_TIME_KEY = "expireTime";
 		final String MAX_ACCOUNT_SIZE = "maxAccSize";
 		final String NICKNAME = "nickname";
```

Comment:
- "/purchased" API should be used to register and update information about subscription
- private boolean connectUserWithOrderId(String userId, String orderId, HttpServletRequest request) {